### PR TITLE
Loads of Sprite Improvements

### PR DESCRIPTION
**Pull request by thevandie**
## About The Pull Request

This adds a LOAD of sprite improvements, from fixing the old colored jumpsuits to giving new sprites to the leather boots and gloves. Also adds a few items to the loadout, to accommodate for their new sprites (for example, I added the colored jumpsuits back to the loadout, and I even added a janitor jumpsuit JUST for you Moon!) This might also be the first ingame addition of Bradhaanata to the Eris, which is incredibly based.

And probably the most influential change, I gave the sneakers a resprite. All of them.

image
image
image
image
image
image
image
image
(I can confirm, bioengies have a green accented labcoat. I was confused at first too).

## Why It's Good For The Game

All of the sprites I changed were old and kinda ugly looking, so a resprite was badly needed. Especially in the case of sneakers, I regret previously suggesting their complete removal.

## Testing

I loaded it up, and every time it broke I fixed it. I also tested the female variants of the clothes and they all worked fine.

## Changelog
:cl:
add: gives new sprites to the colored jumpsuits, and adds them back to the loadout
add: gives new sprites to the sneakers, and adds a new grey one to the loadout
add: gives the psychiatrist's turtleneck a new sprite (in more moemed colors, considering they exist for mental healing)
add: gives the leather shoes a new sprite, and lumps them under the classy shoes selection
add: gives the botany gloves a new sprite, and adds them to the loadout
add: makes the oberth boots use the colorscheme used in their item sprite
add: gives the bio-engie and chemist labcoats new sprites
add: slightly improves the scientist labcoat
add: adds a new blue business casual uniform, and adds it to the loadout under leisure suits selection
add: adds new sprite for the botanist jumpsuit, and adds it to the loadout
add: adds a new sprite for the virologist's uniform, and adds it to the loadout (as it's unused)
add: adds a new sprite for the janitor's jumpsuit, and adds it to the loadout under the jumpsuits selection
add: gives the blue blazer a new sprite, and adds it to the suits selection
add: gives the guild uniform (the beige shirt and jeans) a new sprite
add: gives the guild merchant a new coat sprite (specifically designed to be a remaster of the old guild tech coat that everyone loves)
add: adds the old merchant coat to the loadout, as well as the old technician coat. they are "hand-me-downs" from the merchant
add: changes the guild tech coat SLIGHTLY, to make it more palatable (the newer one everyone hated on)
fix: gives the black softcap it's PROPER sprite (it previously had the cargo softcap sprite)
/:cl:

### CODE REVIEW

File: code/modules/clothing/under/miscellaneous.dm [hunk 1]
```diff
@@ -210,6 +210,12 @@ obj/item/clothing/under/leisure/pullover
 	icon_state = "joe"
 	item_state = "joe"
 
+/obj/item/clothing/under/leisure/casual/blue
+	name = "blue casual suit"
+	desc = "an aqua colored casual suit for office workers."
```
Comment (thevandie) [inline code/modules/clothing/under/miscellaneous.dm:215]: oh shit I'm gonna get wacked for that aren't I? I'll fix it...

File: code/modules/client/preference_setup/loadout/lists/suits.dm [hunk 2]
```diff
@@ -133,3 +133,13 @@
...
+
+/datum/gear/suit/oldguild
+	display_name = "old black guild coat"
+	path = /obj/item/clothing/suit/storage/cargo_jacket/old
+	allowed_roles = list(JOBS_CARGO)
+
+/datum/gear/suit/oldmerchant
+	display_name = "old brown guild coat"
+	path = /obj/item/clothing/suit/storage/qm_coat/old
+	allowed_roles = list(JOBS_CARGO)
```
Comment (thevandie) [inline code/modules/client/preference_setup/loadout/lists/suits.dm:145]: Wait... shit I'm gonna get dinged for that, HOLD ON

Comment (thevandie) [reply code/modules/client/preference_setup/loadout/lists/suits.dm]: (I forgot to add a space at the end, it's added now)

File: code/modules/clothing/suits/jobs.dm [hunk 3]
```diff
@@ -76,7 +76,7 @@
 //Merchant
 /obj/item/clothing/suit/storage/qm_coat
 	name = "guild merchant coat"
-	desc = "An ideal choice for a smuggler. This coat seems have good impact resistance, and is made from resistant and expensive materials."
+	desc = "A fabulous black coat, perfect for a steeled capitalist."
```
Comment (SirRichardFrancis) [inline code/modules/clothing/suits/jobs.dm:79]:
```suggestion
	desc = "Fabulous black coat, perfect for a steeled capitalist."
```
Leaving out "A" and "An" at the start of item descriptions seems to be a pattern, let's keep things consistent here.
Can't 100% tell how grammatically correct it is though.

File: code/modules/clothing/suits/jobs.dm [hunk 4]
```diff
@@ -91,6 +91,12 @@
 	)
 	siemens_coefficient = 0.8
 
+/obj/item/clothing/suit/storage/qm_coat/old
+	name = "old brown guild coat"
+	desc = "One of the merchant's old coats, passed down to the technicians after he was able to purchase a better selection."
```
Comment (SirRichardFrancis) [inline code/modules/clothing/suits/jobs.dm:96]:
```suggestion
	desc = "One of the old merchant coat designs, passed down to technicians after it fell out of fashion."
```
Same as above. Let's not force unnecessary story on every merchant.

File: code/modules/clothing/under/jobs/civilian.dm [hunk 5]
```diff
@@ -25,8 +25,8 @@
 	item_state = "b_suit"
 
 /obj/item/clothing/under/rank/cargotech
-	name = "cargo worker's jumpsuit"
-	desc = "A pair of jeans and turtleneck worn by the cargo workers."
+	name = "guild worker's jumpsuit"
+	desc = "A pair of jeans and turtleneck worn by the guild workers."
```
Comment (SirRichardFrancis) [inline code/modules/clothing/under/jobs/civilian.dm:29]:
```suggestion
	desc = "Pair of jeans and turtleneck worn by the guild workers."
```
See previous comments. May or may not be a good idea, take with a grain of salt.

File: code/modules/clothing/under/jobs/civilian.dm [hunk 6]
```diff
@@ -88,14 +88,14 @@
 	item_state = "b_suit"
 
 /obj/item/clothing/under/rank/hydroponics
-	desc = "A jumpsuit designed to protect against minor plant-related hazards."
+	desc = "A jumpsuit from the Bradhaanata Agroponics 'Farm Yourself' kit."
```
Comment (SirRichardFrancis) [inline code/modules/clothing/under/jobs/civilian.dm:91]:
```suggestion
	desc = "Jumpsuit from the Bradhaanata Agroponics 'Farm Yourself' kit."
```
See above.

File: code/modules/clothing/under/jobs/civilian.dm [hunk 7]
```diff
@@ -88,14 +88,14 @@
...
 	name = "botanist's jumpsuit"
 	icon_state = "hydroponics"
 	item_state = "g_suit"
 	permeability_coefficient = 0.50
 
 /obj/item/clothing/under/rank/janitor
-	desc = "It's the official uniform of the ship's janitor."
+	desc = "An ancient janitorial jumpsuit."
```
Comment (SirRichardFrancis) [inline code/modules/clothing/under/jobs/civilian.dm:98]:
```suggestion
	desc = "Ancient janitorial jumpsuit."
```
See above.

File: code/modules/clothing/under/jobs/medsci.dm [hunk 8]
```diff
@@ -64,8 +64,8 @@
 	)
 
 /obj/item/clothing/under/rank/virologist
-	desc = "It's made of a special fiber that gives special protection against biohazards. It has a virologist rank stripe on it."
-	name = "moebius virologist's jumpsuit"
+	desc = "An old green shirt and khakis, originally worn by the Moebius Virology Corps, now it is discarded in maintenance after their disbandment."
+	name = "moebius virologist's uniform"
```
Comment (SirRichardFrancis) [inline code/modules/clothing/under/jobs/medsci.dm:68]:
```suggestion
	name = "moebius virologist's uniform"
	desc = "Old green shirt and khakis, originally worn by the Moebius Virology Division, now it is discarded in maintenance after their disbandment."
```
Imo shouldn't be in the game at all.
Far as I know Moe got 'divisions' instead of 'corps'.
Also `name` variable usually comes before `desc`. That will have no ill effects on the game, but having variables in order they have been defined is good in terms of code style.

File: code/modules/clothing/under/miscellaneous.dm [hunk 9]
```diff
@@ -157,8 +157,8 @@
 
 
 /obj/item/clothing/under/blazer
-	name = "blue blazer"
-	desc = "A bold but yet conservative outfit, red corduroys, navy blazer and a tie."
+	name = "black tailcoat"
+	desc = "A bold but yet conservative outfit, red slacks and an almost victorian style tailcoat."
```
Comment (SirRichardFrancis) [inline code/modules/clothing/under/miscellaneous.dm:161]:
```suggestion
	desc = "Bold, yet conservative outfit; Red slacks and almost Victorian style tailcoat."
```
See above.

File: code/modules/clothing/under/miscellaneous.dm [hunk 10]
```diff
@@ -210,6 +210,12 @@ obj/item/clothing/under/leisure/pullover
 	icon_state = "joe"
 	item_state = "joe"
 
+/obj/item/clothing/under/leisure/casual/blue
+	name = "blue casual suit"
+	desc = "An aqua colored casual suit for office workers."
```
Comment (SirRichardFrancis) [inline code/modules/clothing/under/miscellaneous.dm:215]:
```suggestion
	desc = "Aqua colored casual suit for office workers."
```

**Review by SirRichardFrancis — CHANGES_REQUESTED** [review]
Virology shirt looks way too bright even for Eris.
Besides, virology is not a thing anymore.

We also don't have a hydroponist role, garden is under the church control for many years now, they have drip of their own.
What viro/hydro uniform is currently in the code is a leftover, which is to be removed, not expanded upon.

**Comment by SirRichardFrancis** [comment]
Also please put pictures and comment to them in PR description, not changelog.

File: code/modules/clothing/under/jobs/medsci.dm (hunk 8 above)
Comment (SirRichardFrancis) [reply code/modules/clothing/under/jobs/medsci.dm:68]: Tbh, virology should have been a part of Medical Division, rater than something independent.

**Comment by Firefox13** [comment]
I'd like some screenshots with how these look on Eris. I'm concerned about bright green jumpsuit/shoes being a little TOO bright. And merchant's coat should stay as-is to stand out from guild techs attire.
~~why half of our jumpsuits are baggy like this even~~

**Comment by thevandie** [comment]
> We also don't have a hydroponist role, garden is under the church control for many years now, they have drip of their own. What viro/hydro uniform is currently in the code is a leftover, which is to be removed, not expanded upon.

The idea was to add more fun shit vagabonds and other crew members could wear, hence why it's in the loadout
Plus, the botany jumpsuit shows up in the club and moesci, the idea was that they just bought farming kits from Bradhaanata for the expedition. I would modify or remove the one from the NT public garden, but that would require mapping and there's already a mapping pr out this week
But yeah I'll dull the viro shirt

File: code/modules/clothing/suits/jobs.dm [hunk 11]
```diff
@@ -57,8 +57,8 @@
 	)
 
 /obj/item/clothing/suit/storage/cargo_jacket/old
-	name = "black guild technician jacket"
-	desc = "Stylish jacket lined with pockets. It seems to have a little protection from physical harm. This one is done in dark black color."
+	name = "old black guild coat"
```
[the remark this answers is not in the document]
Comment (thevandie) [reply code/modules/clothing/suits/jobs.dm:60]: The code stuff is meaningless on thar front
Plus it'd require a lot of path changing and that's annoying

File: code/modules/clothing/suits/jobs.dm (hunk 3 above)
Comment (thevandie) [reply code/modules/clothing/suits/jobs.dm:79]: I thought that was the correct consistency, from looking at other items

**Comment by thevandie** [comment]
image

File: code/modules/clothing/under/jobs/medsci.dm (hunk 8 above)
Comment (thevandie) [reply code/modules/clothing/under/jobs/medsci.dm]: The idea was that it was a DIVISION of the Medical DEPARTMENT
I suppose, but I didn't really think about the logistics of that, it's just defunct now, so here's the uniform.
I plan to board up viro and turn it into a fun set piece in the future, cause right now it's NOT for some reason.

**Comment by thevandie** [comment]
@Firefox13 

> I'd like some screenshots with how these look on Eris. I'm concerned about bright green jumpsuit/shoes being a little TOO bright. And merchant's coat should stay as-is to stand out from guild techs attire.

image

It is done.
I see your point, the sneakers might be a bit too bright for random civilians to have, considering it uses a moemed white for the toes... I'll look into making it darker.

**Comment by Mycah142** [comment]
TL;DR

**Comment by thevandie** [comment]
@Mycah142 
TLDR so far, Kirov wanted me to make nonsensical grammar changes and I committed them in an absence of thought, now I'm reverting them. The Virologist uniform needed darkening so I darkened it. Firefox wanted a better screenshot so I provided it, and now I might need to make the sneakers darker. That's the TLDR.

**Comment by SirRichardFrancis** [comment]
> @SirRichardFrancis 👀
> everything says "A" or "An"
> Imma revert that, man

> Kirov wanted me to make nonsensical grammar changes and I committed them in an absence of thought, now I'm reverting them.

"Everything", yeeeah, right.

There is a lot of things like that all over the place. Names start uppercase (without there being a faction name or similar exception to lowercase rule), descriptions start lowercase, grammar is all messed up, and many more.

Similar story with `var/` thing in arguments of a proc.
Coding guidelines says it shouldn't be there, but nobody bothered to go and retroactively change thousands of procs.
So people continue to copypaste old code with these and then argue "but it's all over the place, I'm not following standards entire codebase doesn't", see original melee rework for a good example.

And unlike code standards, naming stuff is not even written down, works on "maybe someone notices and speaks up" basis.

That said, I'm not able to stop you, and unless a maintainer tells you to do something with a PR - you don't have to.

**Comment by ValoTheValo** [comment]
> Virology shirt looks way too bright even for Eris. Besides, virology is not a thing anymore.
> 
> We also don't have a hydroponist role, garden is under the church control for many years now, they have drip of their own. What viro/hydro uniform is currently in the code is a leftover, which is to be removed, not expanded upon.

don't remove viro uniform. it should remain trash gear as it fits well there.

**Comment by thevandie** [comment]
> "Everything", yeeeah, right.

Considering EVERYTHING in the loadout, and 90% of all the other items I've looked at from SEVERAL different files all put an A or An before the description, I'm gonna go out on a limb and say it's the status quo. The only time it changes is if the description is a lot shorter, or if it's a special case. I can grab examples if you REALLY wanna see.

> That said, I'm not able to stop you, and unless it's a maintainer who tells you to do something with a PR - you don't have to listen.

Well, I know I just blew you off, but in general you're a senior member of the community (and previously a high up developer from what I knew?) so I WANT to be able to trust your judgements, but this is something I have evidence to disprove you on, and I'm concerned about it.
In short, I respect you.

**Comment by Kurgis** [comment]
Nice to see the colored jumpsuits given a needed once-over.

**Comment by thevandie** [comment]
I would like to note, if I've been pinged on discord to change something I physically cannot see it as I do not have access to discord and won't for a couple months.

**Comment by thevandie** [comment]
woops misclick

**Comment by hyperioo** [comment]
Except the convention matter for the descriptions and loadouts that would need a correction pass on the whole codebase to fix the inconsistencies, spires are noice 👍
